core-bundle/contao/models/PageModel.php

```diff
@@ -1165,7 +1165,7 @@ public function getPreviewUrl($strParams=null)
 		$baseUrl = $context->getBaseUrl();
 
 		// Add the preview script
-		$context->setBaseUrl(\dirname($baseUrl) . $previewScript);
+		$context->setBaseUrl(rtrim(\dirname($baseUrl), '/') . $previewScript);
```

Comment:
This prevents URLs like `//preview.php`

Reply:
This can never happen, can it? https://3v4l.org/VI2Sh

Reply:
Yes it is for the case of $baseUrl being `/preview.php`.
This causes the issue that the pages preview URLs all redirect to `//preview.php` currently.